pictor/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+
+  apache:
+    container_name: pictor-apache
+    image: ${ENV_APACHE_CONTAINER} 
+    ports:
+      - "80:80"
+      - "443:443"
+    environment:
+      - ENV_APACHE_SERVER_NAME=${ENV_APACHE_SERVER_NAME}
+    volumes:
+      - ${ENV_REACT_APPDIR}/build:/var/www/html
+      - ${ENV_SSL_CERT_DIR}:/certs
+      - ${ENV_SHIB_CONF_DIR}/shibboleth2.xml:/etc/shibboleth/shibboleth2.xml
+      - ${ENV_WSI_FILES_DIR}:/data/deepZoomImages
```

Comment:
The ENV_WSI_FILES_DIR and ENV_WSI_ORIG_FILES_DIR can both go away.

Reply:
I am also pretty sure we don't need /certs mapped anywhere ( we don't do this for orion ).  I am not sure why we do it for delphinus/circinus, though I suspect it has to do with the modified apache image to handle the multiple apps.

Reply:
Yeah, it's all being brought over as defaults.